src/api/artifact/controller.go

```diff
@@ -56,6 +56,8 @@ type Controller interface {
 	Delete(ctx context.Context, id int64) (err error)
 	// Tags returns the tags according to the query, specify the properties returned with option
 	Tags(ctx context.Context, query *q.Query, option *TagOption) (total int64, tags []*Tag, err error)
+	// CreateTag creates a tag
+	CreateTag(ctx context.Context, tag *Tag) (id int64, err error)
 	// DeleteTag deletes the tag specified by tagID
 	DeleteTag(ctx context.Context, tagID int64) (err error)
```

Comment:
Is this method really delete the tags or just `remove` the tags from the artifact?

Reply:
It really deletes the tag